src/measures.tex

```diff
@@ -5,36 +5,34 @@ \subsection{Spaces of Measures}
   \lean{real_measures}
   \leanok
   Let $0 < p' < 1$ be a real number, and let $S$ be a finite set.
-  Then $\Rm(S)$ denotes the real vector space
-  \[
-	  \Rm(S) = \left\{\sum_{s\in S}a_{s}[s] \text{ such that } a_{s}\in \R\right\}
-  \]
-	endowed with the $\ell^{p'}$-norm $\Vert \sum_{s\in S}a_s[s]\Vert_{\ell^{p'}}=\sum_{s\in S}\lvert a_s\rvert ^{p'}$.
-
-  For every finite set $S$, the space $\Rm(S)$ can be written as the union
-  \[
-    \Rm(S) = \varinjlim_{c>0}\Rm(S)_{\leq c}
-  \]
-  where $\Rm(S)_{\leq c} = \left\{F \in \Rm(S) \text{ such that } \Vert F \Vert_{\ell^{p'}}\leq c\right\}$.
-  Now, given a profinite set $S=\varprojlim S_i$, where all $S_i$'s are finite sets, and a positive real number $c>0$, define $\Rm(S)_{\leq c}$ as
-  \[
-    \Rm(S)_{\leq c}=\varprojlim \Rm(S_i)_{\leq c}
-  \]
-  endowed with the projective limit topology.
-  Finally, set $\Rm(S)=\varinjlim_{c}\Rm(S)_{\leq c}$.
+  Denote by $\Rm(S)$ the real vector space
+\[
+\Rm(S)=\left\{\sum_{s\in S}a_{s} \text{ such that } a_{s}\in \R\right\}
+\]
+endowed with the $\ell^{p'}$-norm $\Vert \sum_{s\in S}a_s\Vert_{\ell^{p'}}=\sum_{s\in S}\lvert a_s\rvert ^{p'}$. We refer to its elements as (real) $\ell^{p'}$-measures on $S$.
 \end{definition}
+For every finite set $S$, the space $\Rm(S)$ can be written as the filtered colimit
```

Comment:
I think I would just call it a union
```suggestion
For every finite set $S$, the space $\Rm(S)$ can be written as the union
```

Reply:
Well, on this I would disagree. In my opinion, in the language of LTE it is more natural to consider it as a colimit, expecially because we need to use its relationship with limits. But of course you decide at the end of the day, so if you stick to your view I will be happy to change.

Reply:
Ok, fair enough. How about
```suggestion
For every finite set $S$, the space $\Rm(S)$ can be written as the filtered colimit (in this case, a union)
```